console/style.rst

```diff
@@ -42,26 +42,22 @@ which allow to create *semantic* commands and forget about their styling.
 Basic Usage
 -----------
 
-In your command, instantiate the :class:`Symfony\\Component\\Console\\Style\\SymfonyStyle`
-class and pass the ``$input`` and ``$output`` variables as its arguments. Then,
-you can start using any of its helpers, such as ``title()``, which displays the
-title of the command::
+In your `__invoke` method, type the :class:`Symfony\\Component\\Console\\Style\\SymfonyStyle`
+argument. Then, you can start using any of its helpers, such as ``title()``, which
```

Comment:
I didn't understand at first time.

Maybe "In your `__invoke` method, replace arguments by one argument of type :class:`Symfony\\Component\\Console\\Style\\SymfonyStyle`" ?


Do you know if we tell somewhere, which typehints are "magically" inejected ? https://github.com/yceruto/symfony/blob/cf6c3aaba8c59c258c16309547540b3dd03aa722/src/Symfony/Component/Console/Command/InvokableCommand.php#L101-L107

Reply:
>Do you know if we tell somewhere, which typehints are "magically" inejected ?

I think yes